pkg/controller/certificaterequests/acme/acme.go

```diff
@@ -161,12 +161,9 @@ func (a *ACME) Sign(ctx context.Context, cr *v1alpha2.CertificateRequest, issuer
 
 	// If the acme order has failed then so too does the CertificateRequest meet the same fate.
 	if acme.IsFailureState(order.Status.State) {
-		message := fmt.Sprintf("Failed to wait for order resource %s/%s to become ready",
-			expectedOrder.Namespace, expectedOrder.Name)
-		err := fmt.Errorf("order is in %q state", order.Status.State)
-
+		message := fmt.Sprintf("Failed to wait for order resource %q to become ready", expectedOrder.Name)
```

Comment:
No longer logging the order namespace...I guess because the event will always be in the same namespace as the cr, order, certificate....but does this message also show up in the cert-manager logs....where the namespace might be useful?

Reply:
This will already be present as part of the structured log message (due to this line here: https://github.com/jetstack/cert-manager/blob/ec44d8992cda4a02f2d014b75ef472aa4970a2b6/pkg/controller/certificaterequests/controller.go#L191)